addons/disco-like-workflows/hooks/post_gen_project.py

```diff
@@ -0,0 +1,55 @@
+import os
```

Comment:
It could be nice to comment what this script is doing ?

Reply:
I will add some docs